### DISCUSSION TITLE
Problem instantiating checkout

### DISCUSSION THREAD

**Discussion by VinceValence** [mercadopago/sdk-js#70] [General]
When I try to create a checkout instance with this code using a valid PUBLIC_KEY:

```html
    <!DOCTYPE html>
    <html>
    <body>
        <div class="cho-container"></div>
        <script src="https://sdk.mercadopago.com/js/v2"></script>
        <script>
            const mp = new MercadoPago('PUBLIC_KEY', {
                locale: 'en-US'
            })
            const checkout = mp.checkout({
                preference: {
                    id: 'YOUR_PREFERENCE_ID'
                }
            });
            checkout.render({
                container: '.cho-container',
                label: 'Pay'
            });
        </script>
    </body>
</html>
```
and I'm getting `"MercadoPago.js - There was an error creating a new checkout instance"`

Could this be a problem with MercadoPago? There seem to be problems with mercadopago according to the [status page](https://status.mercadopago.com/)

**Comment by lucmantovani** [comment]
Hello @VinceValence!
I did some tests locally and couldn't reproduce the error
Are the preferences being created successfully at the backend?

**Reply by VinceValence** [reply]
I am quite sure. I'm making a POST request to here: `https://api.mercadopago.com/checkout/preferences` and it responds with an id. I am using that Id in 

```javascript
const checkout = mp.checkout({
    preference: {
        id: 'HERE'
    }
});
```

**Reply by lucmantovani** [reply]
So it may really be a problem with our checkout...
Could you send more information, like the network tab from the developer tools?
Another question: are there any logs related to the `autoOpen` parameter?

**Reply by VinceValence** [reply]
It started working some minutes ago. I did not change anything at all. 

Regarding dev tools, I didn't look at it when It was not working (sorry, it just did not come to my mind).

Another warning regarding using `autoOpen`  and not `open` appeared when I tried to click the Pay button. 

Since I do not have the problem any more, that's all I can provide.

Thank you very much for taking the time to respond!

**Reply by lucmantovani** [reply]
Ok! I'm glad it was just some temporary instability
Should you have any trouble in future, don't hesitate in getting in contact!
